### PR TITLE
Plants card

### PR DESCRIPTION
**Pull request by silviadmgz**
- Loading skeleton for plant page: please review it as I'm not entirely happy with the dimensions 
- Add location and last watered by
- Reinsert image

### CODE REVIEW

File: frontend/src/pages/Plant/index.tsx [hunk 1]
```diff
@@ -43,20 +50,35 @@ const PlantPage = () => {
...
+			updatedPlantData
+		);
 		toast.success(`${plant?.name} has been successfully watered`);
 		setPlant(response);
 	};
 
+	const viewLoadingSkeleton = () => {
+		return (
+			<div className='flex flex-col pl-8 gap-4 w-full'>
+				<div className='skeleton h-80 w-80 pl-12'></div>
```
Comment (dila1001) [inline frontend/src/pages/Plant/index.tsx:73]: this should have bg-base-200

File: frontend/src/pages/Plant/index.tsx [hunk 2]
```diff
@@ -43,20 +50,35 @@ const PlantPage = () => {
...
+				<div className='skeleton h-16 w-44 pl-12'></div>
```
Comment (dila1001) [inline frontend/src/pages/Plant/index.tsx:74]: make this bg-base-200 too

File: frontend/src/pages/Plant/index.tsx [hunk 3]
```diff
@@ -43,20 +50,35 @@ const PlantPage = () => {
...
+				{[1, 2, 3, 4].map((item) => (
+					<div key={item} className='skeleton w-44 h-8 bg-base-200'></div>
```
Comment (dila1001) [inline frontend/src/pages/Plant/index.tsx:76]: I experimented with the width and i think w-60 looks more natural bcs rn the width is the same as the title skeleton

File: frontend/src/pages/Plants/index.tsx [hunk 4]
```diff
@@ -18,6 +20,9 @@ const PlantsPage = () => {
 		const fetchData = async () => {
 			if (householdId) {
 				const plants = await getPlants(householdId);
+				await getHousehold(householdId).then((name) => {
```
Comment (dila1001) [inline frontend/src/pages/Plants/index.tsx:23]: For the household name, you dont need to fetch. Just get from the households context. Filter by id and get the name.

File: frontend/src/pages/Plant/index.tsx [hunk 5]
```diff
@@ -43,20 +50,35 @@ const PlantPage = () => {
...
-			uploadedImage: plant!.uploadedImage,
+			imageUrl: plant!.imageUrl,
+			// uploadedImage: plant!.uploadedImage,
 			wateringFrequencyInDays: plant!.wateringFrequencyInDays,
 			lastWatered: getTodaysDate(),
 			waterAmountInMl: plant!.waterAmountInMl,
-			imageUrl: plant!.imageUrl,
 			location: plant!.location,
-			lastWateredBy: plant!.lastWateredBy
+			lastWateredBy: plant!.lastWateredBy,
```
Comment (dila1001) [inline frontend/src/pages/Plant/index.tsx:59]: last watered by should be user.firstName

**Review by dila1001 — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Plant/index.tsx (hunk 3 above)
Comment (silviadmgz) [reply frontend/src/pages/Plant/index.tsx]: done :)

**Pull request merged**